### ISSUE TITLE
Need to quote "group" column when selecting in processes table; possible to rename?

### ISSUE THREAD

**Issue by Happy-Dude** [facebook/osquery#1878]
Hey everyone,

When I run `SELECT group FROM processes`, I see the following error: `Error: near "group": syntax error`.

Chatting in the Slack-general channel, it's likely that without quoting/ escaping the keyword, it is interpreting it as the GROUP function keyword in SQL.

To work around that, `SELECT "group" FROM processes` works. Should there be consideration into renaming the group column to something else?

**Comment by theopolis** [comment]
Sure! What do you think is best? Maybe a `pgroup`?

**Comment by Happy-Dude** [comment]
Thanks for the quick reply, @theopolis. I'm thinking "process_group" should work. 

While a bit wordy, it's consistent with how the columns in other tables (*_group) are named. proc_group just doesn't sound as nice.

**Comment by theopolis** [comment]
I agree, `proc_group` doesn't sound good, but why don't you like `pgroup`? It's not wordy and has consistent diction with other projects. The only other use of 'scoped' groups in the osquery specs is `sig_group` for yara, and that's completely artificial so don't let it influence you too much.

**Comment by Happy-Dude** [comment]
Ah, I gotcha.
`pgroup` sounds good, let's go with it!
